config/kubernetes-sigs/sig-api-machinery/teams.yaml

```diff
@@ -104,14 +104,6 @@ teams:
     description: admin access to kubebuilder
     members:
     - camilamacedo86
-    - DirectXMan12
-    - droot
-    - estroz
-    - grodrigues3
-    - Liujingfang1
-    - mengqiy
-    - monopole
-    - pmorie
     - pwittrock
```

Comment:
@mrbobbytables should we not keep here only who is the project owner?

The following ones I know are folks who may have context on ancient history but are no longer directly involved

  - directxman12
  - droot
  - mengqiy
  - estroz